tests/store/tracking/test_file_store.py

```diff
@@ -724,21 +759,63 @@ def test_get_deleted_runs(self):
         assert len(deleted_runs) == 1
         assert deleted_runs[0] == run_id
 
+    @staticmethod
+    def _assert_create_run_appends_to_artifact_uri_path_correctly(cases):
+        for artifact_root_uri, expected_artifact_uri_format in cases:
+            with TempDir() as tmp:
+                fs = FileStore(tmp.path(), artifact_root_uri)
+                exp_id = fs.create_experiment("exp")
+                run = fs.create_run(
+                    experiment_id=exp_id, user_id="user", start_time=0, tags=[], run_name="name"
+                )
+                cwd = Path.cwd().as_posix()
+                drive = Path.cwd().drive
+                if (
+                    platform.system().lower() == "windows"
+                    and expected_artifact_uri_format.startswith("file:")
+                ):
+                    cwd = f"/{cwd}"
+                    drive = f"{drive}/"
+                assert run.info.artifact_uri == expected_artifact_uri_format.format(
+                    e=exp_id, r=run.info.run_id, cwd=cwd, drive=drive
+                )
+
+    @pytest.mark.skipif(os.name != "nt", reason="This test only passes on Windows")
```

Comment:
Should we use the same conditional logic for path parsing for the test exclusion? 
```
platform.system().lower() == "windows"
```